week-5/diego_jonathan_deyvi/agency_tour.sql

```diff
@@ -0,0 +1,37 @@
+
+
+create table Customer
+(
+  id serial PRIMARY key,
+  c_name varchar,
+  c_city varchar,
+  c_state varchar,
+  c_country VARCHAR
+);
+
+create table Product
+(
+  id serial primary key,
+  p_name VARCHAR,
+  p_days INTEGER,
+  p_price INTEGER
+);
+
+create table agency
```

Comment:
Standardize your table names. A recommendation: don't capitalize any of them

Reply:
Cool, I'll keep it in mind 👀